collect_battery_parameters.py

```diff
@@ -0,0 +1,9 @@
+from collect_out_of_range_parameters import *
+from collect_out_of_range_parameters import *
+from check_limits import *
+
+def collect_battery_parameters(parameters_action,parameters_alert,parameter_name,parameter_value,parameter_limit,lang):
+   if lang == 'DE':

```

Comment:
You can pass `lang` directly as the index. The if isn't required.

Reply:
Done!!!

![image](https://user-images.githubusercontent.com/13776900/111645179-78197080-8826-11eb-9328-40403b7e5d2c.png)

Reply:
Done!!!

![image](https://user-images.githubusercontent.com/13776900/111645516-c62e7400-8826-11eb-85a7-118388d932b4.png)